recipe/meta.yaml

```diff
@@ -44,9 +41,7 @@ test:
   commands:
     - pip check
     - jupytext --help
-    - test -f ${PREFIX}/share/jupyter/nbextensions/jupytext/index.js                                              # [unix]
     - test -f ${PREFIX}/share/jupyter/labextensions/jupyterlab-jupytext/package.json                              # [unix]
```

Comment:
We may be able to remove the other two similar commands as well, as I can not find how they are used.

It seems we could use pytest-cov and I also see [this](https://github.com/mwouts/jupytext/blob/v1.16.1/pyproject.toml#L155), which seems slightly related but it still makes me believe they can be removed at first glance, but I'm interested to hear yours and others thoughts on this.

Reply:
I believe it's testing to see if this file exists https://github.com/mwouts/jupytext/blob/v1.16.1/jupyterlab/package.json

Reply:
Sorry, I tagged the wrong line. 🤦‍♀️ I meant [this line](https://github.com/mwouts/jupytext/blob/v1.16.1/pyproject.toml#L150)

Reply:
My only opinion is that some tests are better than none, and this is the way conda forge is doing it.